src/Symfony/Component/Serializer/Encoder/XmlEncoder.php

```diff
@@ -60,6 +60,7 @@ class XmlEncoder implements EncoderInterface, DecoderInterface, NormalizationAwa
     public const VERSION = 'xml_version';
     public const CDATA_WRAPPING = 'cdata_wrapping';
     public const CDATA_WRAPPING_PATTERN = 'cdata_wrapping_pattern';
+    public const IGNORE_EMPTY_ATTRIBUTES = 'ignore_empty_attributes';
```

Comment:
Could you update the `XmlEncoderContextBuilder` conscequently?

Reply:
done